src/MagnumPlugins/DdsImporter/DdsImporter.cpp

```diff
@@ -143,6 +285,160 @@ void swizzlePixels(const PixelFormat format, Containers::Array<char>& data) {
+        case DxgiFormat::R16G16UInt:
+            return {PixelFormat::RGInteger, PixelType::UnsignedShort};
+        case DxgiFormat::R16G16Float:
+            return {PixelFormat::RG, PixelType::HalfFloat};
+        case DxgiFormat::R16G16UNorm:
+            return {PixelFormat::RG, PixelType::UnsignedShort};
+        case DxgiFormat::R16G16SNorm:
+            return {PixelFormat::RG, PixelType::Short};
+        case DxgiFormat::R16G16SInt:
+            return {PixelFormat::RGInteger, PixelType::Short};
+
+        /* R32 and D32 Formats */
+        case DxgiFormat::R32Typeless:
+        case DxgiFormat::R32UInt:
+            return {PixelFormat::RedInteger, PixelType::UnsignedInt};
+        case DxgiFormat::R32Float:
+            return {PixelFormat::Red, PixelType::Float};
+        case DxgiFormat::R32SInt:
+            return {PixelFormat::RedInteger, PixelType::Int};
+        case DxgiFormat::D32Float:
+            return {PixelFormat::DepthComponent, PixelType::Float};
+
+        /* RG8 Formats */
+        case DxgiFormat::R8G8Typeless:
+        case DxgiFormat::R8G8UInt:
+            return {PixelFormat::RGInteger, PixelType::UnsignedByte};
+        case DxgiFormat::R8G8UNorm:
+            return {PixelFormat::RG, PixelType::UnsignedByte};
+        case DxgiFormat::R8G8SNorm:
+            return {PixelFormat::RG, PixelType::Byte};
+        case DxgiFormat::R8G8SInt:
+            return {PixelFormat::RGInteger, PixelType::Byte};
+
+        /* R16 and D16 Formats */
+        case DxgiFormat::R16Typeless:
+        case DxgiFormat::R16UInt:
+            return {PixelFormat::RedInteger, PixelType::UnsignedShort};
+        case DxgiFormat::R16Float:
+            return {
+                #ifndef MAGNUM_TARGET_GLES2
+                PixelFormat::Red
+                #else
+                PixelFormat::Luminance
+                #endif
+                , PixelType::HalfFloat};
+        case DxgiFormat::D16UNorm:
+            return {PixelFormat::DepthComponent, PixelType::UnsignedShort};
+        case DxgiFormat::R16UNorm:
+            return {
+                #ifndef MAGNUM_TARGET_GLES2
+                PixelFormat::Red
+                #else
+                PixelFormat::Luminance
+                #endif
+                , PixelType::UnsignedShort};
+        case DxgiFormat::R16SNorm:
+            return {
+                #ifndef MAGNUM_TARGET_GLES2
+                PixelFormat::Red
+                #else
+                PixelFormat::Luminance
+                #endif
+                , PixelType::Short};
+        case DxgiFormat::R16SInt:
+            return {PixelFormat::RedInteger, PixelType::Short};
+
+        /* R8 and A8 Formats */
+        case DxgiFormat::R8Typeless:
+        case DxgiFormat::R8UInt:
+            return {PixelFormat::RedInteger, PixelType::UnsignedByte};
+        case DxgiFormat::A8UNorm:
+            //return {PixelFormat::Alpha, PixelType::UnsignedByte}; //TODO:Alpha? Using same as R8Unorm
```

Comment:
GL treats both as single-channel (`PixelFormat::Red`) without any semantic distinction. There was Luminance, Alpha and LuminanceAlpha once, but were deprecated in favor of those that don't have any implicitly associated meaning. The app should know what given texture contains.

I would make it the same as `R8Unorm` but put a detailed chapter into the docs about such mapping "exceptions".